datadog_checks_base/datadog_checks/base/ddyaml.py

```diff
@@ -90,14 +94,25 @@ def safe_yaml_load(stream, Loader=yLoader):
 
 def safe_yaml_load_all(stream, Loader=yLoader):
     if Loader != yLoader:
-        log.warning("Unsafe loading of YAML has been disabled - using safe loader instead")
+        stream_name = get_stream_name(stream)
+        log.info("Unsafe loading of YAML has been disabled - using safe loader instead in {}".format(stream_name))
 
     if pyyaml_load_all:
         return pyyaml_load_all(stream, Loader=yLoader)
 
     return yaml.load_all(stream, Loader=yLoader)
 
 
+def get_stream_name(stream):
+    """Using the same logic as pyyaml to handle both string types and file types"""
+    if isinstance(stream, string_types):
+        return "<string>"
+    elif hasattr(stream, 'name'):
+        return realpath(stream.name)
+    else:
+        return "<file>"
```

Comment:
Is it safe to assume it's a file `<file>` here ? Based on your comment above, is it what pyyaml is already doing ?

Maybe it worth adding a comment near `return "<file>"` to help the readers.

Reply:
@FlorianVeaux Any news on this feedback? :)

Reply:
^ The docstring comment has been made more explicit, is that enough?

Reply:
My bad, didn't saw the change.

👍